tests/Authorization.AspNetCore.Tests/ValidationTestBase.cs

```diff
@@ -92,7 +93,8 @@ private IValidationResult Validate(ValidationTestConfig config)
             Document = document,
             Operation = document.Definitions.OfType<GraphQLOperationDefinition>().First(),
             Rules = config.Rules,
-            Variables = config.Variables
+            Variables = config.Variables,
+            RequestServices = ServiceProvider,
```

Comment:
`ServiceProvider` is root service provider. Maybe create and set service provider from scope ?